src/app/pages/Zufallsgenerator/Zufallsgenerator.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import NavBar from "../../components/NavBar/NavBar";
+import BigMovieCard from "../../components/BigMovieCard/BigMovieCard";
+import Button from "../../components/Button/Button";
+import Footer from "../../components/Footer/Footer";
+import styles from "./Zufallsgenerator.module.css";
+
+function Zufallsgenerator(): JSX.Element {
+  return (
+    <div className={styles.container}>
+      <main>
+        <div>
+          <NavBar />
+        </div>
```

Comment:
as this div only includes the NavBar maybe consider changing it into a <nav> instead as it is more precise